DigitalOffice/Shared/Kernel/CustomExceptionHandler.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Net;
+using System.Threading.Tasks;
+using Kernel.Exceptions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Diagnostics;
+using System.Text.Json;
+
+namespace Kernel
+{
+    /// <summary>
+    /// Represents exception handler middleware. Provides method for handle exceptions.
+    /// </summary>
+    public static class CustomExceptionHandler
+    {
+        /// <summary>
+        /// Handle exceptions.
+        /// </summary>
+        /// <param name="context">Specified http context.</param>
+        /// <returns>Asynchronous operation for handling exceptions.</returns>
+        public static async Task HandleCustomException(HttpContext context)
```

Comment:
Usually add an async at the end of an async method. Not?

Reply:
Agreed, most of built-in methods from Microsoft or EFCore have this, so its a good idea